hokusai/staging.yml

```diff
@@ -93,6 +93,17 @@ spec:
                 operator: In
                 values:
                 - foreground
+                - foreground-spot
```

Comment:
allow pods to run on nodes [labeled with foreground-spot](https://github.com/artsy/substance/blob/4809569bb58fd86f20eab2ce513ccd253dde4435/clusters/staging/federation/kubernetes-staging-leo.artsy.systems/spec.yml#L183).